Extensions/Physics2Behavior/Box2D_v2.3.1_min.wasm.js

```diff
@@ -7,20 +7,22 @@ function(Box2D) {
   Box2D = Box2D || {};
 
 
-var a;a||(a=typeof Box2D !== 'undefined' ? Box2D : {});var aa;a.ready=new Promise(function(b){aa=b});var ba={},ca;for(ca in a)a.hasOwnProperty(ca)&&(ba[ca]=a[ca]);var da=!1,ea=!1,fa=!1,ha=!1;da="object"===typeof window;ea="function"===typeof importScripts;fa="object"===typeof process&&"object"===typeof process.versions&&"string"===typeof process.versions.node;ha=!da&&!fa&&!ea;var ia="",ja,ka,la,ma;
+var a;a||(a=typeof Box2D !== 'undefined' ? Box2D : {
+  locateFile: function(path, prefix) {
+    return location.protocol === 'file' ?
+        // This is needed to run in preview.
```

Comment:
```suggestion
        // This is needed to run on Electron.
```

Reply:
I tested a windows build and there is not the same issue as for preview. Does it use Electron too?

Reply:
Alright, weird 🤔 A windows build is definitely Electron based, you can look at the installed files! 
Would you be able to find what "location.protocol" is returning? I want to be 100% sure of what this is.

Reply:
I tried again a Windows build withe the new version and it's working. But, I don't have access to the console so I don't know which protocol is used.
I've probably not tested is correctly the 1st time.